rr_common/include/msg_srv_names.hpp

```diff
@@ -10,12 +10,28 @@
 
 #include <string.h>
 
+static std::string rr= "/rr_vehicle/";
```

Comment:
Me and @DongJunJin fleshed out some updates to how this common topic/service names file will work, but you've got the right idea here! I'll make changes to this to reflect that